hs_core/templates/pages/genericresource.html

```diff
@@ -2359,6 +2374,39 @@ <h3 style="color: red" id="file-type-error"></h3>
             }
         });
 
+        var resource_id = $("#resource_id").attr("value");
+
+        // for tracking when a user is selected in add creator/contributor model form
+        $(".deck").bind("DOMNodeInserted",function(){
```

Comment:
`DOMNodeInserted` and `DOMNodeRemoved` are both deprecated and have severe performance problems in the browsers that do support them.

Instead, handle these within the code where you are inserting/removing these items in the first place. The coupling via DOM is fragile.